CMakeLists.txt

```diff
@@ -1,9 +1,66 @@
 ﻿# CMakeList.txt : CMake project for VocalTractLabApi, include source and define
 # project specific logic here.
 #
-cmake_minimum_required (VERSION 3.8)
+cmake_minimum_required (VERSION 3.13)
```

Comment:
> Looks good generally, but I do not like the way `include/` is split up into `Backend` and `VocalTractLabApi`, since the source files are not split up either. In any way, a more fitting name for the api files would be `API_C`, as we may add a c++ api in the future. For the backend, I'd prefer lowercases,` backend`.

Splitting up the header files into Backend and VocalTractLabApi folders is best practice to avoid ambiguities. Let's say we were to use some other library at some point that included a header file with a name that already exists in the backend or API. If the header files were at the root level of `include/`, there was no way to unambiguously choose the right one to include. Using the subfolders, you just set the `include/` directory and then include like so:

```cpp
#include "Backend/Glottis.h"
#include "SomeOtherLib/Glottis.h
```

(This of course brings us to attention the fact that we don't have namespaces in our codespace, but that is a topic for another time)

Reply:
The uppercase `Backend` is conforming to Peter's conventions. As is the name `VocalTractLabApi`.

Reply:
And yes, there was a reason why CMake 3.13 is required now, but I'm gonna have to check which routine made that necessary.

Reply:
> 

I get that, but why are the source files not split up, its the same situation isn't it?

Reply:
The FetchContent module requires version 3.11+ and since 3.13 source paths can be relative, which makes listing the sources in the subfolders much tidier. I think that was why.

Reply:
> The uppercase Backend is conforming to Peter's conventions. As is the name VocalTractLabApi.

So how would you name an additional C++ API? Besides, I disagree, because that is only the name of the dir, the name of the file remains unchanged.

Reply:
> 

That is a valid point. In fact, the API should actually provide another interface to the static Backend lib. I'm going to restructure that.

Reply:
The C++ API is the Backend lib, isn't it?

Reply:
Thats the static lib, but we may add a dynamic lib in the future as well.

Reply:
BTW, other point: The old speaker file may be removed from `resources/`

Reply:
> 
That would still be VocalTractLabBackend.dll or *.so, then.

Reply:
Hm, maybe we should change the name of the Backend folder(s) to `VocalTractLabBackend`? Seems more consistent compared to the API folders.

Reply:
> Hm, maybe we should change the name of the Backend folder(s) to VocalTractLabBackend? Seems more consistent compared to the API folders. And just Backend seems very generic.

Yes, VocalTractLabBackend sounds better.

Reply:
> That would still be VocalTractLabBackend.dll or *.so, then.

True, you are right.

Reply:
Okay, I thought that we could simply wrap the VocalTractLabApi header around the static VocalTractLabBackend lib and build a shared lib from it, but apparently it is not as straight-forward. The recommended way of doing it is to include all the backend sources again explicitly when building the shared API library.